kivy_ios/toolchain.py

```diff
@@ -176,6 +176,19 @@ def get_env(self):
         include_dirs += ["-I{}".format(
             join(self.ctx.dist_dir, "include", self.name))]
 
+        # Add Python include directories
+        include_dirs += [
+            "-I{}".format(
+                join(
+                    self.ctx.dist_dir,
+                    "root",
+                    "python3",
+                    "include",
+                    "python3.11",
```

Comment:
should this be retrieved dynamically? Same question for kivy_ios/recipes/hostpython3/__init__.py

Reply:
Looks like it was pretty easy in both places. Fixed it.